.github/workflows/ci-v6.yml

```diff
@@ -0,0 +1,33 @@
+# This is displayed in the pr check area
+name: Comparison to 6.X.X
+
+# Controls when the action will run. Triggers the workflow on pull request
+# events but only for the 5.X.X branch
+on:
+    pull_request:
+        branches: [6.X.X]
```

Comment:
Du kannst auch einfach in der existierenden Datei einfach diese Änderung vornehmen

Reply:
ich weiß, aber so wären getrennte ci features möglich. siehe chat